.cspell.json

```diff
@@ -92,6 +92,7 @@
     "testname",
     "Tobio",
     "tsbuildinfo",
+    "uzair",
```

Comment:
?

Reply:
Ohh I added this before I knew about Cspell:ignore

I will remove this in the next commit 
"uzair" is my name actually